src/constants.js

```diff
@@ -364,5 +364,22 @@ export const KEYS = {
     enter: 'Enter',
 };
 
+/* ------------------ Javascript Object Types ----------------- */
```

Comment:
You can isolate these constants to a file in the component directory if they're not intended to be used elsewhere.